examples/karpenter/main.tf

```diff
@@ -195,6 +232,7 @@ module "vpc" {
   private_subnets = [for k, v in local.azs : cidrsubnet(local.vpc_cidr, 8, k + 10)]
 
   enable_nat_gateway   = true
+  create_igw           = true
```

Comment:
this is the [default](https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/variables.tf#L1146) so not needed

Reply:
Ok